### PR TITLE
bugfix: profile photo on account page

### LINKED ISSUE #768: [BUG] Fix bugs of profile photo on the Accout page
**Bug №1**

- Change word `Перетягніть` into "ти"-form => `Перетягни`;
- Coordinate words `Перетягни` and `Обрати` => `Перетягни` and `Обери`.

**Actual result:**

Image

**Bug №2**

Cross (X) placed unequally to the edges of the window.

**Actual result:**

Image

Image

Image

**Expected result ([figma](https://www.figma.com/file/FHBiKogMc9VzGWojaQxA9c/%F0%9F%92%96fictadvisor?type=design&node-id=10446%3A72152&mode=design&t=uyEJldzbFEt8OIKY-1)):**

Image

**Bug №3**

In the error after choosing file with wrong extension as profile photo: 

- Word "файлу" written twice;
- Word "Підтримувані" written incorrectly.
- Error must be written in English;
- Сross placed incorrectly.

**Steps to reproduce:**

- Click on your profile photo;
- Click on "Обрати файл" button;
- Choose file with wrong extension;
- Pay attention to the error message.

Image

Environment:
• Deploy: Dev;
• URL: https://dev.fictadvisor.com/account;
• OS: Windows 10;
• Browser: Google Chrome 116.0.5845.163.

### PR DESCRIPTION
**Pull request by sifes**
closed #768

### CODE REVIEW

File: src/components/common/ui/alert/Alert.styles.ts [hunk 1]
```diff
@@ -49,9 +49,16 @@ export const alert = (
       height: '16px',
     },
   },
-
   '.MuiAlert-action': {
     cursor: 'pointer',
+    color: 'grey.800',
```
Comment (Yevheniy2020) [inline src/components/common/ui/alert/Alert.styles.ts:54]: color: theme.palette.grey[800]

File: src/components/common/ui/alert/Alert.styles.ts [hunk 2]
```diff
@@ -49,9 +49,16 @@ export const alert = (
...
+    margin: '0 0 0 0',
```
Comment (Yevheniy2020) [inline src/components/common/ui/alert/Alert.styles.ts:55]: margin: 0

**Review by Yevheniy2020 — APPROVED** [review]

**Comment by kol-oss** [comment]
qa approved

File: src/components/pages/account-page/components/general-tab/components/change-avatar-window/components/avatar-dropzone/utils/handleFileSelect.ts [hunk 3]
```diff
@@ -11,9 +11,9 @@ export const handleFileSelect = (
 ) => {
   if (!isValidFile(file)) {
     toast.error(
-      'Неправильне розширення файлу файлу',
-      'Підтримуванні розширення: .png, .jpg, .jpeg, .webp',
-      4000,
+      'Invalid file extension',
+      'Supported file extensions: .png, .jpg, .jpeg, .webp',
+      2000000,
```
Comment (Markusplay) [inline src/components/pages/account-page/components/general-tab/components/change-avatar-window/components/avatar-dropzone/utils/handleFileSelect.ts:16]: make time normal as it was

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Pull request merged**